Source/JavaScriptCore/bytecode/ExecutionCounter.h

```diff
@@ -55,7 +55,7 @@ inline int32_t formattedTotalExecutionCount(float value)
 
 template<CountingVariant countingVariant>
 class ExecutionCounter {
-    WTF_MAKE_TZONE_ALLOCATED_TEMPLATE(ExecutionCounter);
+    WTF_FORBID_NEW_DELETE_OPERATORS(ExecutionCounter);
```

Comment:
I think we should name this something like `WTF_FORBID_HEAP_ALLOCATION_WITH_VALIDATION` and add a FIXME to replace `WTF_FORBID_HEAP_ALLOCATION` with this. Alternatively, we could rename `WTF_FORBID_HEAP_ALLOCATION` to `WTF_FORBID_HEAP_ALLOCATION_DEPRECATED` with a comment to switch to this and name this `WTF_FORBID_HEAP_ALLOCATION`